Call_while_GC.txt

```diff
@@ -0,0 +1,18 @@
+safe functions:
```

Comment:
We don't want this file in the final version, right?

Reply:
Oops... It was for my use only. It was never meant to be part of the PR. Sorry.